GeoLib/AnalyticalGeometry.h

```diff
@@ -123,6 +148,19 @@ void rotatePoints(MathLib::DenseMatrix<double> const& rot_mat, std::vector<GeoLi
 void rotatePointsToXY(std::vector<GeoLib::Point*> &pnts);
 
 /**
+ * rotate points to X-Y plane
+ * @param p_pnts_begin Iterator pointing to the initial element in a vector of points used for computing a rotation matrix
+ * @param p_pnts_end Iterator pointing to the final element in a vector of points used for computing a rotation matrix
```

Comment:
Do you mean an iterator to the element following the last element of the vector? Same for line 155.